march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -444,20 +449,18 @@ void MarchHardwareInterface::updateIMotionCubeState()
 
 void MarchHardwareInterface::iMotionCubeStateCheck(size_t joint_index)
```

Comment:
```suggestion
bool MarchHardwareInterface::iMotionCubeStateCheck(size_t joint_index)
```
Also make this change in the header file